template/patches/react-native+0.66.5.patch

```diff
@@ -1,10 +1,23 @@
+diff --git a/node_modules/react-native/ReactCommon/yoga/yoga/Yoga.cpp b/node_modules/react-native/ReactCommon/yoga/yoga/Yoga.cpp
+index 2c68674..ea95e40 100644
+--- a/node_modules/react-native/ReactCommon/yoga/yoga/Yoga.cpp
++++ b/node_modules/react-native/ReactCommon/yoga/yoga/Yoga.cpp
+@@ -2229,7 +2229,7 @@ static float YGDistributeFreeSpaceSecondPass(
+         depth,
+         generationCount);
+     node->setLayoutHadOverflow(
+-        node->getLayout().hadOverflow() |
++        node->getLayout().hadOverflow() ||
```

Comment:
This is the new fix - I think the rest of the diff is just from getting shifted around in the `patch-package` codegen process.

Reply:
Nice. Looks like this was fixed in a later version of react native, but never backported back to 0.66.5. (https://stackoverflow.com/a/75898723) Thanks for catching it!